### PR TITLE
feat: add menu for managing events and log out

### PR DESCRIPTION
**Pull request by nikit19**
First steps of #27 
Manage events opens the organizer's app if it is installed or else opens up the organizer's app in playstore.
Shifted log out to menu.
screenshot_20180601-202442

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt [hunk 1]
```diff
@@ -68,6 +66,61 @@ class ProfileFragment : Fragment() {
...
+
+    fun startOrgaApp(packageName: String) {
+        try {
+            val intent = Intent("android.intent.action.MAIN")
+            intent.addCategory("android.intent.category.LAUNCHER")
+
+            intent.addFlags(Intent.FLAG_ACTIVITY_NO_ANIMATION)
+            val resolveInfoList = activity?.packageManager?.queryIntentActivities(intent, 0)
+
+            for (info in resolveInfoList.orEmpty())
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt]: Add curly braces around

File: app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt [hunk 2]
```diff
@@ -68,6 +66,61 @@ class ProfileFragment : Fragment() {
...
+                if (info.activityInfo.packageName.equals(packageName, ignoreCase = true)) {
+                    launchComponent(info.activityInfo.packageName, info.activityInfo.name)
+                    return
+                }
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt]: This is too much logic. Just let the app launch and handle exception

File: app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt [hunk 3]
```diff
@@ -68,6 +65,44 @@ class ProfileFragment : Fragment() {
...
+        menu?.setGroupVisible(R.id.profile_menu, true)
+        super.onPrepareOptionsMenu(menu)
+    }
+
+    private fun startOrgaApp(packageName: String) {
+        val manager = activity?.packageManager
+        try {
+            val i = manager?.getLaunchIntentForPackage(packageName)
+                    ?: throw  ActivityNotFoundException()
+            i.addCategory(Intent.CATEGORY_LAUNCHER)
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt]: `intent`

**Comment by nikit19** [comment]
PR updated. Please review

**Comment by iamareebjamal** [comment]
@nikit19 Please now focus on non auth features

**Pull request merged**